### ISSUE TITLE
New suggestion for /shells/8.3.0/debugging-shell-commands.html

### ISSUE THREAD

**Issue by ksaper** [QualiSystems/devguide_source#167]
when setting a resource's attributes, if the type a cloudshell password, in the driver we are most likely using an API call to decrypt the value for use (like an Enable Password) - here we enter it as plain text, which then will get scrambled because we "decrypt" it.

**Comment by TomerAdmon** [comment]
Thanks for the feedback @ksaper, 
Are you referring to a specific section/example in the page? to the "Debugging the shell" maybe?

Tomer

**Comment by ksaper** [comment]
Yes in the example for the Debugging the Shell - where we use the magic mock to setup the driver test/debugger
